integration/basic_test.go

```diff
@@ -127,3 +127,18 @@ func TestFindCommentQuery(t *testing.T) {
 	require.NoError(t, err)
 	assert.Contains(t, databaseNames, name)
 }
+
+func TestLongCollectionName(t *testing.T) {
```

Comment:
I like this test! Should we make it a big more generic and cover a few cases (long names, short names)?

Reply:
In general there are 3 possible cases: less than 63 symbols, 63 symbols, more than 63 symbols.
First one is already covered by all other integration tests as they are creating collections. Third case covered with this test. I think we can add the second case.

Reply:
Added.